java/ql/src/semmle/code/java/security/XSS.qll

```diff
@@ -30,6 +31,11 @@ class XssSink extends DataFlow::ExprNode {
         m.getAReference().getArgument(1) = this.getExpr() and m.getName() = "loadDataWithBaseURL"
       )
     )
+    or
+    exists(ReturnStmt rs |
```

Comment:
Should we check that the method's return type is String or byte[], or similar? I expect that Spring would serialize Objects to `json` or `xml` form (or other) depending on the content type. Such serializers should take care of escaping the values of fields where needed.